pom.xml

```diff
@@ -63,6 +63,7 @@
     </pluginRepositories>
     <properties>
         <jenkins.version>1.642.3</jenkins.version>
+        <jenkins-test-harness.version>2.15-SNAPSHOT</jenkins-test-harness.version>
```

Comment:
Downstream of https://github.com/jenkinsci/jenkins-test-harness/pull/32. Actually passes without this dependency, but takes three times as long to collect the reference, due to lack of exponential escalation in the original version.